source/shared-geo/src/test/java/nl/overheid/aerius/geo/shared/BBoxTest.java

```diff
@@ -52,4 +52,12 @@ public void testIsPointWithinBoundingBox() {
     point2.setY(RECEPTOR_BBOX.getMaxY());
     Assertions.assertTrue(RECEPTOR_BBOX.isPointWithinBoundingBox(point2), "maximum should be included");
   }
+
+  @Test
+  void testEquals() {
+    final BBox TEST_BBOX_EQUAL = new BBox(3604, 296800, 287959, 629300);
```

Comment:
styling of local variables (otherwise you'll get new sonar warnings :smile:)
```suggestion
    final BBox testBBoxEqual = new BBox(3604, 296800, 287959, 629300);
```